deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# build and push images
+printenv DOCKER_PASSWORD | docker login --username "$DOCKER_USERNAME" --password-stdin
+docker build -t codeforpoznan/pah-fm-frontend:latest frontend
+docker build -t codeforpoznan/pah-fm-backend:latest  backend
+docker push codeforpoznan/pah-fm-frontend
+docker push codeforpoznan/pah-fm-backend
+
+# build and push statics
+(cd frontend && npm     run       build                    && cp -r dist   ../public)
+(cd backend  && python3 manage.py collectstatic --no-input && cp -r static ../public)
+aws s3         sync                public s3://codeforpoznan-public/dev_pah_fm
+aws cloudfront create-invalidation --paths "/*" --distribution-id E2ESZ1QPNV00X
+
+# bundle application
+pip install --quiet -r backend/requirements/base.txt --target packages
+(cd packages && rm -r psycopg2 && svn checkout https://github.com/jkehler/awslambda-psycopg2/trunk/psycopg2-3.6 && mv psycopg2-3.6 psycopg2)
```

Comment:
it's ugly but it works, psycopg2 has some problems under the hood, that's an easy way of dealing with them